css-brochure-form/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <h1>LEARN MORE</h1>
+  <form class="container">
+    <div>
+      <label class="label-class">COMPANY EMAIL</label>
+      <input class="email" name="email" placeholder="sally@company.co">
```

Comment:
We are still missing some attributes on both labels and inputs throughout the page in order to connect those together